---
title: インポート エラー – 関係 0 は現在、データをインポートするように設定されていません
description: 説明
solution: Analytics
product: Analytics
applies-to: Analytics
keywords: KCS
resolution: Resolution
internal-notes: null
bug: false
article-created-by: Krunal Oza
article-created-date: 10/19/2022 6:14:16 PM
article-published-by: Krunal Oza
article-published-date: 10/19/2022 6:17:12 PM
version-number: 6
article-number: KA-16999
dynamics-url: https://adobe-ent.crm.dynamics.com/main.aspx?forceUCI=1&pagetype=entityrecord&etn=knowledgearticle&id=aab9e5d1-d94f-ed11-bba2-00224808679b
exl-id: 70458c4d-2e67-458f-90ce-df9234b9cb70
source-git-commit: 9cffbfb76a10a760a40f56f9ecccdcc16ee61872
workflow-type: tm+mt
source-wordcount: '99'
ht-degree: 9%

---

# インポート エラー – 関係 0 は現在、データをインポートするように設定されていません

## 説明

<b>環境</b>
- Customer Journey Analytics
- Analytics



<b>問題/症状</b>
Classifications （SAINT） API で Classifications API インポートエラー「Relation 0 は、現在、データをインポートするように設定されていません」が発生する理由を教えてください。


## 解決策


変数「」に、「関係 0 は、現在、データを読み込むように設定されていません」というエラーが表示されます&#x200B;*製品*」は、で使用されます *大文字「Product」*. Classifications API は、で「p」を持つ変数のみを受け取ります。 *小文字* as ``*製品*」と入力します。
